src/build.py

```diff
@@ -549,21 +552,26 @@ def SyncArchive(out_dir, name, url):
       with open(stamp_file) as f:
         stamp_url = f.read().strip()
       if stamp_url == url:
-        print '%s directory already exists' % name
+        print('%s directory already exists' % name)
         return
-    print '%s directory exists but is not up-to-date' % name
-  print 'Downloading %s from %s' % (name, url)
+    print('%s directory exists but is not up-to-date' % name)
+  print('Downloading %s from %s' % (name, url))
   work_dir = os.path.dirname(out_dir)
 
   try:
-    f = urllib2.urlopen(url)
-    print 'URL: %s' % f.geturl()
-    print 'Info: %s' % f.info()
+    if sys.version_info.major == 2:
+      from urllib2 import urlopen, URLError
+    else:
+      from urllib.request import urlopen, URLError
```

Comment:
Why if/else here but  try/except for `queue`

Maybe the if/else is better because it self documenting?

Reply:
Yeah I switched the other one for that reason.